src/python/pants/task/console_task.py

```diff
@@ -29,7 +29,7 @@ def register_options(cls, register):
 
   def __init__(self, *args, **kwargs):
     super(ConsoleTask, self).__init__(*args, **kwargs)
-    self._console_separator = self.get_options().sep.decode('unicode_escape')
+    self._console_separator = self.get_options().sep.encode('ascii').decode('unicode_escape')
```

Comment:
The order matters here. I originally had `encode('unicode_escape').decode('ascii')` and it resulted in Pants outputting `\\\\n` instead of `\n`.